sdk/exporter/exporter.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporter
+
+import (
+	"context"
+	"sync"
+	"sync/atomic"
+)
+
+// Type defines all the types of exporters we have available
+type Type int32
+
+// All the exporter types
+const (
+	ExporterTypeSync = iota
+	ExporterTypeBatch
+)
+
+// SyncExporter is a synchronous exporter
+type SyncExporter interface {
+	ExportSpan(context.Context, interface{})
```

Comment:
I've added a context here, as was suggested in the examples on #77. There is never a context available when we call this method however.
This might come later, but it still seems a bit weird.

Reply:
keep the context for now.